src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/VisualEditModePageObject.java

```diff
@@ -110,6 +112,11 @@ public VisualEditModePageObject() {
     super();
   }
 
+  public VisualEditModePageObject clickBoldButton() {
+    boldButton.click();
```

Comment:
in general it's good to add `wait.forElementClickable()` before performing `click()` action